folio-token/src/test/java/org/folio/spring/systemuser/SystemUserServiceTest.java

```diff
@@ -44,12 +46,14 @@ class SystemUserServiceTest {
   public static final String OKAPI_URL = "http://okapi";
   private static final String TENANT_ID = "test";
   private static final Instant TOKEN_EXPIRATION = Instant.now().plus(1, ChronoUnit.DAYS);
+  private static final String MOCK_TOKEN = "eyJhbGciOiJIUzI1NiJ9eyJzdWIiOiJ0ZXN0X2FkbWluIiwidXNlcl9pZCI6ImQyNjUwOGJ"
+      + "lLTJmMGItNTUyMC1iZTNkLWQwYjRkOWNkNmY2ZSIsImlhdCI6MTYxNjQ4NDc5NCwidGVuYW50IjoidGVzdCJ9VRYeA0s1O14hAXoTG34EAl80";
```

Comment:
I don't think you want this to be an actual token. Even though it might not be, it very much looks like storing a secret in source code, which is a big red flag. Just make the token any string. It shouldn't matter.

Reply:
done as per the suggestion